### PR TITLE
[FIX JENKINS-33822]

### PR DESCRIPTION
**Pull request by gusreiber**
- made radio buttons visible
- set first radio on blur from name
- allow tab index to pass through copy without setting it to the selected
- but if there is text in the copy field, show it as selected and set it selected

@reviewbybees

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
infra issue, retriggering the build

**Comment by recena** [comment]
[JENKINS-33822](https://issues.jenkins-ci.org/browse/JENKINS-33822)

File: war/src/main/js/widgets/add/addform.less [hunk 1]
```diff
@@ -63,39 +63,26 @@
     position: relative;
     border: 1px solid #cccccc;
     border-top: none;
-    padding: 20px;
     background-color: #f9f9f9;
+    padding:10px;
```
Comment (recena) [inline war/src/main/js/widgets/add/addform.less]: :bug: unrelated change

File: war/src/main/js/widgets/add/addform.less [hunk 2]
```diff
@@ -63,39 +63,26 @@
...
 
     p.description {
       font-size: 16px;
-      margin-top: 0;
+      margin:0;
+      padding: 10px 10px 0;
```
Comment (recena) [inline war/src/main/js/widgets/add/addform.less]: :bug: unrelated change

File: war/src/main/js/widgets/add/addform.less [hunk 3]
```diff
@@ -63,39 +63,26 @@
...
     }
 
-    .add-item-copy {
-      position: relative;
-      padding-left: 58px;
+    input[type=radio] {
+      position:absolute;
+      left:-99999px;
+    }
```
Comment (recena) [inline war/src/main/js/widgets/add/addform.less]: Please, follow the current convention:
```
input[type=radio] {
  position: absolute;
  left: -99999px;
}
```

File: war/src/main/js/widgets/add/addform.less [hunk 4]
```diff
@@ -115,8 +102,19 @@
...
+    .add-item-copy{
+      label{
+        display:inline-block;
+        font-size:1em;
+        line-height: 48px;
+        padding: 0 10px 0 0; 
+        font-weight:normal;     
+      }      
+    }
+  }
```
Comment (recena) [inline war/src/main/js/widgets/add/addform.less]: :bug: to mix coding styles

File: war/src/main/js/widgets/add/addform.less [hunk 5]
```diff
@@ -304,12 +302,9 @@
     -o-transition: .5s;
     transition: .5s;
 
-    .icon {
-      opacity: 1
-    }
```
Comment (recena) [inline war/src/main/js/widgets/add/addform.less]: :bug: unrelated change

File: war/src/main/js/widgets/add/addform.less [hunk 6]
```diff
@@ -334,8 +329,7 @@
       input {
         position: absolute;
         top: 24px;
-        left: 25px;
-        display: none;
+        left: -99925px;
```
Comment (recena) [inline war/src/main/js/widgets/add/addform.less]: :bug: no needed since you has defined a more specific definition above:

```
input[type=radio] {
  position: absolute;
  left: -99999px;
}
```

File: war/src/main/js/add-item.js [hunk 7]
```diff
@@ -82,6 +96,25 @@ $.when(getItems()).done(function(data){
       $('.input-help', context).removeClass('input-message-disabled');
     }
 
+    function setSelectState(e) {
```
Comment (recena) [inline war/src/main/js/add-item.js]: :ant: The param `e` is never used

File: war/src/main/js/widgets/add/addform.less [hunk 8]
```diff
@@ -281,14 +279,14 @@
   position: relative;
 }
 
-.j-item-options {
-
-  li:hover, li:focus {
+#createItem {
+  .add-item-copy:hover, .add-item-copy:focus,
+  .categories li:hover, li:focus {
```
Comment (recena) [inline war/src/main/js/widgets/add/addform.less]: :bug: unrelated change

Comment (recena) [inline war/src/main/js/widgets/add/addform.less]: Don't make sense to consider the *option* copy at the same level than an item. Unrelated with the problem described in [JENKINS-33822](https://issues.jenkins-ci.org/browse/JENKINS-33822)

**Comment by recena** [comment]
By the way, In Firefox 45.0.1 on Mac this PR seems do not work. In Chrome 49.0.2623.110 on Mac the behavior it is the same that prior versions of Jenkins, what means **you can not create new items with the keyboard**.

File: war/src/main/js/widgets/add/addform.less (hunk 2 above)
Comment (gusreiber) [inline war/src/main/js/widgets/add/addform.less]: Same as above

File: war/src/main/js/widgets/add/addform.less (hunk 1 above)
Comment (gusreiber) [inline war/src/main/js/widgets/add/addform.less]: This is not an unrelated change. Copy item needs to show selected state, it did not before.

File: war/src/main/js/widgets/add/addform.less (hunk 8 above)
Comment (gusreiber) [inline war/src/main/js/widgets/add/addform.less]: I disagree.

**Comment by gusreiber** [comment]
@recena ? It works for me.

File: war/src/main/js/widgets/add/addform.less (hunk 8 above)
Comment (recena) [inline war/src/main/js/widgets/add/addform.less]: Please send a new PR for this.

File: war/src/main/js/widgets/add/addform.less (hunk 1 above)
Comment (recena) [inline war/src/main/js/widgets/add/addform.less]: It is unrelated because the Jira issue describes clearly what it should be fixed

:-1:

**Comment by oleg-nenashev** [comment]
Marked as WiP according to the discussion above

**Comment by daniel-beck** [comment]
How does this compare to #2250?

**Comment by recena** [comment]
@daniel-beck https://github.com/jenkinsci/jenkins/pull/2250 is an alternative to this one. Both trying to solve the same but with based on different approaches.

**Comment by daniel-beck** [comment]
Merged #2250.
